### ISSUE TITLE
Getting error from downloading module when running terraform init

### ISSUE THREAD

**Issue by jhwang09** [cloudposse/terraform-aws-ecr#31]
I am getting this error when I run against the `master` branch. Currently I am skipping the latest commit and fetch from the second to last commit, it worked as expected. I am suspecting the last commit could have caused the issue 

```
Error downloading modules: Error loading modules: module alert-engine-ecr: Error parsing .terraform/modules/612474be353f3466b76e6152fc91c0be/main.tf: At 2:42: Unknown token: 2:42 IDENT length
```

**Comment by zenbiking** [comment]
Try using a 0.12.x version of terraform and it should work fine.

**Comment by Nuru** [comment]
We strongly recommend never using the `master` branch directly. Always use a specific release. 

The last release to work with Terraform v0.11 is [0.6.1](https://github.com/cloudposse/terraform-aws-ecr/releases/tag/0.6.1). It is unlikely we will do any further work to support Terraform v0.11 but if we do, it will be on the `0.11` branch and will be released as 0.6.x.

Starting with release 0.7.0 Terraform v0.12 is required.
